raiden/raiden_service.py

```diff
@@ -134,6 +136,20 @@ def save_snapshot(serialization_file, raiden):
         )
 
 
+def endpoint_registry_exception_handler(greenlet):
+    try:
+        greenlet.get()
+    except (JSONRPCPollTimeoutException, Exception) as e:
+        if (e.message == "timeout when polling for transaction" or
```

Comment:
`Exception.message` is deprecated and has been removed in Py3. The recommended way to get the exception's message is now either just `str(ex)` or (more explicitly) `ex.args[0]`.